src/app.html

```diff
@@ -5,6 +5,12 @@
     <link rel="icon" href="%sveltekit.assets%/favicon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     %sveltekit.head%
+    <script>
+      var ui_theme = localStorage.getItem("ui_theme");
+      if (ui_theme && ui_theme !== "default") {
+        document.querySelector("html").dataset.theme = ui_theme;
```

Comment:
can we make this sveltey instead of JSy? `data-theme={ui_theme}`. Load ui_theme in a `<script lang="ts">` outside head? P2, but nice.

Also: does this flash on page reload? Haven't tried, but worried about a race setting this.

Reply:
> can we make this sveltey instead of JSy?

I wanted to make it this way first, but then I understood that it's out of any Svelte component (<html> tag). While all the possible components are inside <html> tag itself (even deeper inside - body > div#app > ...). If you know how to make better the code of this, feel free to commit or suggest any code changes. I'm newbie in Svelte and don't know so many things and tricks in Svelte as other people know :)

> Load ui_theme in a <script lang="ts"> outside head?

It can possibly cause flickering when theme is switching while page is loading, so will be anyway more reliable to keep this code inside than in any other place

> P2

What means P2?

> Also: does this flash on page reload?

I was thinked about it and tested. It works without flickering, so I kept using this way and shared with everybody 👌